[@types/plotly.js] Unable to install - invalid tag name

If you know how to fix the issue, make a pull request instead.

- [ X ] I tried using the `@types/xxxx` package and had problems.
- [ ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [ X ] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @chrisgervang @mtadams007 @meganrm @Dadstart 

If you do not mention the authors the issue will be ignored.
-------------------------------------


Trying to install with npm and getting the following error. Sorry if this is a stupid question but it seems like a legitimate issue to me.

> npm install @types/plotly.js

> npm ERR! code EINVALIDTAGNAME
> npm ERR! Invalid tag name "^[object Object]": Tags may not have any characters that encodeURIComponent encodes.

Error while trying to update `@types/plotly.js@1.44.28` to `@types/plotly.js@1.44.31`
```
$ npm update @types/plotly.js
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "^[object Object]": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/usr/.npm/_logs/2020-03-11T19_41_17_265Z-debug.log
```
Environment:
```
$ node -v
v13.1.0
```
```
$ npm -v
6.14.2
```
```
$ npm list @types/plotly.js
...
└── @types/plotly.js@1.44.28
```
```
$ npm outdated
Package                Current   Wanted   Latest  Location
...       
@types/plotly.js       1.44.28  1.44.31  1.44.31 ...
...
```
```
$ ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 9.0.6
Node: 13.1.0
OS: linux x64

Angular: 9.0.6
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.6
@angular-devkit/build-angular     0.900.6
@angular-devkit/build-optimizer   0.900.6
@angular-devkit/build-webpack     0.900.6
@angular-devkit/core              9.0.6
@angular-devkit/schematics        9.0.6
@angular/cdk                      9.1.2
@angular/flex-layout              9.0.0-beta.29
@angular/material                 9.1.2
@ngtools/webpack                  9.0.6
@schematics/angular               9.0.6
@schematics/update                0.900.6
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2
```

--- a/types/plotly.js/index.d.ts
+++ b/types/plotly.js/index.d.ts
@@ -15,6 +15,7 @@
 //                 Takafumi Yamaguchi <https://github.com/zeroyoichihachi>
 //                 Michael Adams <https://github.com/mtadams007>
 //                 Michael Arnett <https://github.com/marnett-git>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 

